src/markd/renderer.cr

```diff
@@ -26,20 +26,14 @@ module Markd
       lit("\n") if @last_output != "\n"
     end
 
-    def escape(text, preserve_entities = false)
```

Comment:
I am not sure if `preserve_entities` served any purpose, it was at least not specced and the result from `gsub` would be the same anyway because only the four special characters are really replaced.

Reply:
You are right.